caproto/_utils.py

```diff
@@ -133,6 +143,10 @@ def get_environment_variables():
             result.get('EPICS_CA_AUTO_ADDR_LIST', '').upper() != 'NO'):
         warn("EPICS_CA_ADDR_LIST is set but will be ignored because "
              "EPICS_CA_AUTO_AUTO_ADDR_LIST is not set to 'no'.")
+        ca_auto_addr_list = result.get('EPICS_CA_AUTO_ADDR_LIST')
+        ca_addr_list = result.get('EPICS_CA_ADDR_LIST')
+        warn(f"EPICS_CA_ADDR_LIST={ca_addr_list!r}")
```

Comment:
The snarky commit message associated with this change is fair. :- )

Can these be moved into the same, single warning though?

Reply:
It's actually more directed at appveyor - despite setting both, we don't see them in `os.environ`. Should try to confirm that when one of us tests it locally.